src/AddTodoForm.jsx

```diff
@@ -21,9 +22,9 @@ const AddTodoForm = ({ onAddTodo }) => {
 
   return (
     <form onSubmit={handleAddTodo}>
-      <label htmlFor="todoTitle">Title: </label>
-      <input type="text" id="todoTitle" name="title" value={todoTitle} onChange={handleTitleChange} required />
-      <input type="submit" value="Add" />
+      <InputWithLabel todoTitle={todoTitle} handleTitleChange={handleTitleChange}>
+        Title
+      </InputWithLabel>
```

Comment:
Change this to reflect the new `InputWithLabel` change

Reply:
Could you please take a look at the changes I made?